src/Kafka/ManageKafkaPermissions/components/AssignPermissionsManual.tsx

```diff
@@ -0,0 +1,156 @@
+import {
+  ResourcePrefixRule,
+  ResourcePrefixRuleValue,
+} from "./ResourcePrefixRule";
+import { ResourceType, ResourceTypeValue } from "./ResourceType";
+import { ShortcutsTableHead } from "./ShortcutsTableHead";
+import { useTranslation } from "react-i18next";
+import { KafkaInstanceManualPermissions } from "./KafkaInstanceManualPermissions";
+
+export type AssignPermissionsManualProps = {
+  resourceType: ResourceTypeValue | undefined;
+  onChangeResourceType: (value: ResourceTypeValue | undefined) => void;
+  submitted: boolean;
+  resourcePrefix: ResourcePrefixRuleValue;
+  onChangeResourcePrefix: (value: ResourcePrefixRuleValue) => void;
+  resourceName: string | undefined;
+  onChangeResource: (value: string | undefined) => void;
+  onFetchResourceNameOptions: (filter: string) => Promise<string[]>;
+  resourcePermission: ResourcePermissionValue;
+  onChangeResourcePermission: (value: ResourcePermissionValue) => void;
+  resourceOperation: ResourceOperationValue | undefined;
+  onChangeResourceOperation: (
+    value: ResourceOperationValue | undefined
+  ) => void;
+  multipleShorctutPermissions?: boolean;
+};
+
+export const AssignPermissionsManual: React.FC<
+  AssignPermissionsManualProps
+> = ({
+  resourceType,
+  onChangeResourceType,
+  submitted,
+  resourcePrefix,
+  onChangeResourcePrefix,
+  resourceName,
+  onChangeResource,
+  onFetchResourceNameOptions,
+  resourcePermission,
+  resourceOperation,
+  onChangeResourceOperation,
+  onChangeResourcePermission,
+  multipleShorctutPermissions = false,
+}) => {
+  const { t } = useTranslation("manage-kafka-permissions");
+  const resourceTypeOptions = () => {
+    if (resourceType === undefined)
+      return [
+        t("operations.all"),
+        t("operations.read"),
+        t("operations.write"),
+        t("operations.create"),
+        t("operations.delete"),
+        t("operations.alter"),
+        t("operations.describe"),
+        t("operations.describe_configs"),
+        t("operations.alter_configs"),
+      ];
+    switch (resourceType) {
+      case "consumer-group":
+        return [
+          t("operations.all"),
+          t("operations.read"),
+          t("operations.delete"),
+          t("operations.describe"),
+        ];
+      case "kafka-instance":
+        return [t("operations.alter"), t("operations.describe")];
+      case "transactional-id":
+        return [
+          t("operations.all"),
+          t("operations.write"),
+          t("operations.describe"),
+        ];
+      default:
+        return [
+          t("operations.all"),
+          t("operations.read"),
+          t("operations.write"),
+          t("operations.create"),
+          t("operations.delete"),
+          t("operations.alter"),
+          t("operations.describe"),
+          t("operations.describe_configs"),
+          t("operations.alter_configs"),
+        ];
+    }
+  };
+  return (
+    <>
+      {!multipleShorctutPermissions ? <ShortcutsTableHead /> : null}
```

Comment:
Header should always visible, no matter single row or multiple rows.

Reply:
I agree, header will be visible at all time. The prop multipeShortcutPermissions is an optional prop and meant only to be used for storybook. When we create the parent component we will simple call <ShortcutsTableHead/> but for now I added this to show that the table alignment will stay in place even if the table header is not present, and to create stories for multiple permissions and only one permission. 
MultipleShortcutsPermission flag will not be used outside storybook and the componenet will be called in the parent on next PR which is in work